t_frang/test_whitelist_mark.py

```diff
@@ -0,0 +1,116 @@
+"""Tests for Frang directive `whitelist_mark`."""
+
+from framework import curl_client, deproxy_client, tester
+from framework.mixins import NetfilterMarkMixin
+from helpers import remote, tempesta, tf_cfg
+
+__author__ = "Tempesta Technologies, Inc."
+__copyright__ = "Copyright (C) 2023 Tempesta Technologies, Inc."
+__license__ = "GPL2"
+
+
+class FrangWhitelistMarkTestCase(tester.TempestaTest, NetfilterMarkMixin):
```

Comment:
Please add these tests to [tests_disabled.json](https://github.com/tempesta-tech/tempesta-test/blob/master/tests_disabled.json).